lib/ansible/cli/galaxy.py

```diff
@@ -301,6 +302,10 @@ def add_list_options(self, parser, parents=None):
 
         list_parser.add_argument(galaxy_type, help=galaxy_type.capitalize(), nargs='?', metavar=galaxy_type)
 
+        if galaxy_type == 'collection':
+            list_parser.add_argument('--format', dest='output_format', choices=('human', 'yaml', 'json'), default='human',
```

Comment:
Do we need to specify `human` as an option? That could just the default and `yaml` or `json` are specified.

Reply:
This is not a blocker, just a thought on the interface.

Reply:
@samdoran This was requested here: https://github.com/ansible/ansible/pull/73474#discussion_r569948199 I have also added a test for this functionality.

 I don't have a strong opinion either way. I will defer to the group. @s-hertel @webknjaz @jborean93 @bcoca - thoughts?

Reply:
I was unable to find examples of a `default` that is not listed in `choices` on the internet. This may need to be checked experimentally.

Reply:
@shanemcd my understanding is that @sdoran suggests `choices=('yaml', 'json'), default='human'` and it's undocumented whether this would work argparse-wise.

Reply:
It _does_ seem to work. Again, I have no strong feelings here. I'm happy to make this change + delete the explicit 'human' format test if that's what folks want.

Reply:
Talked with Sam and he said this is not a blocker. Just going to leave in this way.